tests/fixtures/simple_project/pyproject.toml

```diff
@@ -1,25 +1,25 @@
-[tool.poetry]
-name = "simple-project"
-version = "1.2.3"
-description = "Some description."
-authors = [
-    "Sébastien Eustace <sebastien@eustace.io>"
-]
-license = "MIT"
-
-readme = "README.rst"
-
-homepage = "https://poetry.eustace.io"
-repository = "https://github.com/sdispater/poetry"
-documentation = "https://poetry.eustace.io/docs"
-
-keywords = ["packaging", "dependency", "poetry"]
-
-classifiers = [
-    "Topic :: Software Development :: Build Tools",
-    "Topic :: Software Development :: Libraries :: Python Modules"
-]
-
-# Requirements
-[tool.poetry.dependencies]
-python = "~2.7 || ^3.6"
+[tool.poetry]

```

Comment:
Whats changing in this file?

Reply:
There should not be any changes, looks like somehow Windows line endings slipped in. I'll revert this.